src/environments.js

```diff
@@ -66,10 +66,12 @@ var environmentDefinitions = [
             var parser = this;
             // Currently only supports alignment, no separators like | yet.
             colalign = colalign.value.map ? colalign.value : [colalign];
-            colalign = colalign.map(function(node) {
+            var cols = colalign.map(function(node) {
                 var ca = node.value;
                 if ("lcr".indexOf(ca) !== -1) {
-                    return ca;
+                    return {
+                        align: ca
+                    };
```

Comment:
Yes. Many of the things mentioned in #269 could go in there. `numVerticalLinesAfter` for `|` or `parboxWidth` for `p{…}` or `separatorAfter` for `@{…}` or `intro` and `outro` for `>{…}` and `<{…}`, to name a few ideas.